src/main/java/uk/ac/ebi/eva/utils/MongoDBHelper.java

```diff
@@ -16,42 +16,62 @@
 
 package uk.ac.ebi.eva.utils;
```

Comment:
Maybe split in two? The utility static methods could stay in the `utils` package, and the `getMongoOperations()` and related ones be moved to a new class `uk.ac.ebi.eva.pipeline.configuration.MongoConfiguration`.